py/tests/test_core.py

```diff
@@ -98,15 +103,15 @@ def test_parse_proto_descriptor(self):
         assert name == "Timestamp"
         assert package == "farm_ng.core.timestamp_pb2"
 
-    def test_write_read(self, log_file: Path) -> None:
+    def test_write_read(self, log_file: Path, reader_log_file: Path) -> None:
         num_events = 10
         with EventsFileWriter(log_file) as writer:
             for i in range(num_events):
                 time_stamp = timestamp_pb2.Timestamp(stamp=i)
                 writer.write(path="hello", message=time_stamp)
                 writer.write(path="world", message=time_stamp)
 
-        with EventsFileReader(log_file) as reader:
+        with EventsFileReader(reader_log_file) as reader:
```

Comment:
can you extend this test to cause it to roll over?

Reply:
https://github.com/farm-ng/farm-ng-core/pull/58/commits/78a0d5953ede67ce99e45338c6136c473a518079